scripts/download_wiktionary_test_files.py

```diff
@@ -6,7 +6,8 @@
 import requests
 import os
 from typing import List, Set
-from tests import test
+# pylint: disable=F0401
+from tests import test_core as test
```

Comment:
Any reason why `pylint` has issues with this particular import?

Reply:
I'm unsure. That script works when run in virtualenv and Visual Studio Code intellisense can travel to the import no problem. Only pylint's the one complaining. I'm currently looking if adding it in a new *.env* file would convince pylint that there's no problem.

Reply:
> 
> 
> Any reason why `pylint` has issues with this particular import?

Okay, I just got rid of the comment and even removed the *.env* file with the following content
```
PYTHONPATH=tests/
```
Ran pylint again, and the complaint disappeared. Not sure what that's about.

Reply:
Will this issue prevent the accepting of this pull request?

Reply:
Seems like this issue turns up if `pylint` is not installed in the same directory as python packages
https://github.com/Microsoft/vscode-python/issues/1185

Reply:
Odd, could have sworn that I already set VS to use the *python.exe* in venv. Want me to remove it?

Reply:
> Will this issue prevent the accepting of this pull request?

Not really, I was just curious as to why `pylint` as having a problem with this specific import. I think this comment can be removed then?

Reply:
Got it. Comment removed and commits rebased.

Reply:
> Odd, could have sworn that I already set VS to use the python.exe in venv

Yeah I've faced a fair share of weird VSCode issues as well hahah